sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -121,72 +119,88 @@ void printArgs(Arg0 arg0, Args... args) {
 
 // Utility function to check return from pi calls.
 // Throws if pi_result is not a PI_SUCCESS.
-// TODO: Absorb this utility in Trace Class
-template <typename Exception> inline void piCheckThrow(PiResult pi_result) {
+template <typename Exception = cl::sycl::runtime_error>
+inline void piCheckResult(PiResult pi_result) {
```

Comment:
we decided to not prepend with "pi" utilities in "pi" namespace, so this would be just checkResult, or maybe better check PiResult